app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/Location.java

```diff
@@ -0,0 +1,27 @@
+package ca.ualberta.CMPUT3012019T02.alexandria.model;
+
+public class Location {
+    private double latitude = 0;
+    private double longitude = 0;
+
+    public Location(double latitude, double longitude) {
+        this.latitude = latitude;
+        this.longitude = longitude;
+    }
+
+    public double getLatitude() {
```

Comment:
We could make a Location object that holds both longitude and latitude. Otherwise, the other option is to return a 2 element array/list which is unintuitive in my opinion.

Reply:
whatever works here, I just foresee us having to make to get calls every time we get the location

Reply:
The location currently only lives in Messages in the database, so we get it when we get all the messages. We should be able to pass the lat and lon into the google maps object and it should not make any additional calls, since it just needs to put a pin on the map and it always needs to load some tile